src/main/java/org/schemaspy/model/Catalog.java

```diff
@@ -61,5 +58,16 @@ public int compareTo(Catalog i) {
     public String toString() {
         return name;
     }
-    
+
+	@Override public boolean equals(Object o) {
```

Comment:
Please add test(s) for this new code.
My proposal is to use the brilliant [EqualsVerifier](http://jqno.nl/equalsverifier/) that makes those tests more or less a oneliner.